src/main/java/frc/robot/common/LEDDriver.java

```diff
@@ -0,0 +1,13 @@
+package frc.robot.common;
+
+import edu.wpi.first.wpilibj.Spark;
+
+public class LEDDriver extends Spark {
+
+    public LEDDriver(int port) {
+        super(port);
+    }
+
+    public static final double autonomous = -0.99;
+    public static final double teleop = -0.61;
```

Comment:
constants should be CAPITAL_CASE. 

Out of curiosity, what do these numbers represent

Reply:
My favorite piece of documentation ever has to be Rust's [page on naming conventions](https://doc.rust-lang.org/1.0.0/style/style/naming/README.html), which refers to this as `SCREAMING_SNAKE_CASE`.